src/api/AMOGUS.Infrastructure/DependencyInjection.cs

```diff
@@ -17,16 +17,31 @@ public static class DependencyInjection {
 
         public static IServiceCollection AddDataServices(this IServiceCollection services, IConfiguration configuration) {
 
-            services.AddDbContext<ApplicationDbContext>(options =>
-                options.UseSqlite(configuration.GetConnectionString("DefaultConnection"))
-            );
+            AddDatabaseContext(services, configuration);
             services.AddScoped<IApplicationDbContext>(provider => provider.GetRequiredService<ApplicationDbContext>());
 
-            services.AddIdentity<ApplicationUser, IdentityRole>()
-                .AddEntityFrameworkStores<ApplicationDbContext>()
-                .AddDefaultTokenProviders();
+            AddIdentitiyServices(services);
+            AddAuthenticationServices(services, configuration);
+
+            services.AddTransient<IAuthService, AuthService>();
+            services.AddTransient<IUserService, UserService>();
 
+            return services;
+        }
+
+        private static void AddDatabaseContext(IServiceCollection services, IConfiguration configuration) {
+            services.AddDbContext<ApplicationDbContext>(options =>
+                            options.UseSqlite(configuration.GetConnectionString("DefaultConnection"))
+                        );
+        }
 
+        private static void AddIdentitiyServices(IServiceCollection services) {
```

Comment:
hier auch returntype `IServiceCollection` und parameter `(this IServiceCollection services)`